# creates a new reasoner with the given reasoner id
string[] domain_files
string[] requirements
string[] notify_start
---
uint64 reasoner_id
bool created